website/docs/feature-flag-tutorials/python/implementing-feature-flags.md

```diff
@@ -0,0 +1,346 @@
+
+The modified return statement in this method will now look like this:
+
+```py
+return render_template("surveys_list.html", surveys=surveys, client=client)
+```
+
+In the same file, we will create a new route and a ‘delete’ method with this code snippet:
+
+```py
+@bp.route("/surveys/<int:survey_id>/delete", methods=["GET", "POST", "DELETE"])
+def delete_survey(survey_id):
+   survey = db.get_or_404(Survey, survey_id)
+   db.session.delete(survey)
+   db.session.commit()
+
+   return redirect(url_for("surveys.surveys_list_page"))
+```
+
+The server now has a route that uses a survey ID to locate the survey in the database and delete it.
+
+To make calls to this route, we will create a delete button for each survey on the page.
+
+In `src/backend/templates/surveys_list.html`, add the following code to your survey table underneath the “Survey Page” button:
+
+```html
+{% if client.is_enabled('delete_survey_flag') %}
+<td class="text-end"><a href="{{ url_for('surveys.delete_survey', survey_id=survey.id) }}" class="btn btn-sm btn-danger">Delete</a></td>
+{% endif %}
+```
+
+This code wraps a Delete button in a conditional statement that checks whether or not the feature flag is enabled. This button has a link that points to the `delete_survey` method we created, which will pull in the survey using an ID to search the database, find the matching survey, and delete it from the database session.
+
+Your surveys page will now look something like this:
+
+
+![Screenshot of app in browser with delete buttons in survey table](/img/python-tutorial-surveys-with-delete.png)
+
+
+Test the new functionality by deleting one of your surveys. The page should refresh without the survey you deleted.
+
+
+## 6. Verify the toggle experience
+
+
+Now that we’ve added in new functionality and connected it to our feature flag, we can verify that if you disable the flag in your development environment, your Python app will no longer serve an HTML page with the option to delete surveys you’ve created.
+
+In your local Unleash instance, turn off the feature flag by disabling it in the development environment.
+
+![Image of feature flag with a disabled environment](/img/python-tutorial-disabled-flag.png)
+
+Next, return to your Survey app and refresh the browser. With the flag disabled, the Delete button will no longer be visible.
+
+![Screenshot of app in browser without delete buttons for surveys](/img/python-tutorial-surveys-without-delete.png)
+
+
+## 7. Improve a feature flag implementation with error handling
+
+If you turn the feature flag off, you won’t be able to use the delete button to remove a survey from your list. However, if a user wanted to bypass the UI to delete a survey, they could still use the URL from the delete method route to target a survey and delete it.
+
+You could do this by entering in a URL like this to the browser:
+
+[http://localhost:50505/surveys/2/delete](http://localhost:50505/surveys/2/delete)
```

Comment:
Do you want to provide a linkable example that will open a browser and make db changes to a running application? Sounds a bit dangerous to me!

Also note - if we drop the GET verb I suggested earlier, this won't work!